tests/x11regressions/tracker/tracker_by_command.pm

```diff
@@ -17,12 +17,9 @@ use testapi;
 sub run() {
     my $self = shift;
     x11_start_program("xterm");
-    sleep 2;
-    wait_idle;
     script_run "tracker-search newfile";
-    sleep 5;
     assert_screen 'tracker-cmdsearch-newfile';
-    script_run "exit";
+    send_key 'ctrl-d';
```

Comment:
no, this exit is actually not changing a prompt, but exiting an xterm. I wanted to make that clear it's actually acting on GUI.